osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -232,6 +232,8 @@ public void SelectPreviousRandom()
 
         public bool AllowSelection = true;
 
+        public bool PendingFilter => filterTask?.Completed == false;

```

Comment:
I'd just inline this condition since it's used only in a singnle place. Makes reading the code easier since it becomes more obvious what it's doing.